content/en/docs/installation/operator.md

```diff
@@ -62,51 +25,36 @@ Before you start, ensure the following requirements are met:
 - You have `ValidatingAdmissionWebhook` enabled in the kube-apiserver. Alternatively, you can pass the `--disable-admission-controller` parameter to the to the `deployment.yaml` file that deploys the operator.
 - You have admin rights to install the Custom Resource Definition (CRD) for Operator.
 
-## Accounts CRD (Experimental)
-
-Operator introduces a new CRD for Spinnaker accounts. `SpinnakerAccount` is defined in an object - separate from the main Spinnaker config - so its creation and maintenance can easily be automated.
 
-To read more about this CRD, see [SpinnakerAccount](https://github.com/armory/spinnaker-operator/blob/master/doc/spinnaker-accounts.md).
+## Operator Install
 
-## Install Operator
-
-### Download the Operator Manifests
+Operator has two distinct modes:
+- **Basic**: Operator in basic mode installs Spinnaker into a single namespace without `ValidatingAdmissionWebhook` for doing preflight checks.
```

Comment:
```suggestion
- **Basic**: for installing the Operator and Spinnaker in a single namespace without doing pre-flight checks.
```

Reply:
I don't think anyone reading this will care if it's a validating admissions webhook or not, just that we won't do pre-flight checks before applying the manifest. Thoughts?